src/profile/profile.js

```diff
@@ -3,10 +3,46 @@
 const appendCSS = require('../appendCSS');
 const css = require('./profile.css');
 const template = require('./profile.html').default;
+const axios = require('axios');
```

Comment:
Use `const api = require('../api');` instead of axios directly. That handles switching between dev/prod, and no need to put the base url.